### ISSUE TITLE
[Bug][A11Y][Pin Code Input] Read helper text with the group label

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#487]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

iOS and Android

### App version

0.6.0

### Describe the issue

Helper text is read at last after the inputs

### Expected Behavior

Helper text should be read as a hint after the group label "Enter your 6-digit code", so the user is aware about what is expected before reaching the edit field

**Comment by nouha06** [comment]
@pya35 currently in the hint we hear ‘Double tap to edit’. Should the helper text be announced before or after it?

**Comment by pya35** [comment]
@nouha06 helper text should be announced before

**Comment by pya35** [comment]
OK
